salt/modules/x509.py

```diff
@@ -1138,8 +1138,9 @@ def create_certificate(
     ca_server:
         Request a remotely signed certificate from ca_server. For this to
         work, a ``signing_policy`` must be specified, and that same policy
-        must be configured on the ca_server. See ``signing_policy`` for
-        details. Also the salt master must permit peers to call the
+        must be configured on the ca_server (name or list of ca server).
+        See ``signing_policy`` for details.
+        Also the salt master must permit peers to call the
         ``sign_remote_certificate`` function.
 
         Example:
```

Comment:
Would it be possible to add an example of using multiple CA servers? If not here, at least somewhere it would be good to have an example of the syntax.

Reply:
I added an example with a list in the states x509.py file.